tools/perf_util/pom.xml

```diff
@@ -0,0 +1,56 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.msft</groupId>
+	<artifactId>ort_perf_tool</artifactId>
```

Comment:
ort_perf_tool is not the most descriptive name. how about something like "send_perf_metrics"?

Reply:
i think this rename should be done elsewhere too. e.g., the file paths, the .gitignore entry